axlearn/common/seed_test.py

```diff
@@ -0,0 +1,22 @@
+import os
+from absl.testing import absltest
+from axlearn.common import test_utils
+import tensorflow as tf
+import importlib
+
+
+class SeedTest(test_utils.TestCase):
```

Comment:
where do we actually set the seed?

Reply:
The seed has to be set as an environment variable from any launch script like this:
export DATA_SEED=42
The launch script has not been added to the PR.